pkgs/development/python-modules/astroquery/default.nix

```diff
@@ -6,21 +6,42 @@
 , keyring
 , beautifulsoup4
 , html5lib
+, pytest
+, pytest-astropy
+, astropy-helpers
 }:
 
 buildPythonPackage rec {
   pname = "astroquery";
   version = "0.3.9";
 
-  doCheck = false; # Tests require the pytest-astropy package
-
   src = fetchPypi {
     inherit pname version;
     sha256 = "0zw3xp2rfc6h2v569iqsyvzhfnzp7bfjb7jrj61is1hrqw1cqjrb";
   };
 
+  # Fix tests using conftest.py from HEAD in the upstream GitHub
+  # repository.
+  patches = [ ./conftest.patch ];
```

Comment:
It's probably better to use `fetchpatch` and extract the `.patch` from the actual PR (or just commit) upstream that actually fixed the issue. See other uses of `fetchpatch` - this is a common pattern.

Reply:
This is what I did for other packages, but for this one several commits are needed. So rather than fetching multiple commits, I created a single patch from these.